src/prefect/tasks/secrets/__init__.py

```diff
@@ -7,3 +7,4 @@ class for interacting with other secret providers. Secrets always use a special
 """
 from .base import SecretBase, PrefectSecret, Secret
 from .env_var import EnvVarSecret
+from .vault_secret import VaultSecret
```

Comment:
I think you'll have to remove this so the user will have to import from the full path if they want vault secrets since it uses an extra requirement.

Reply:
vault_secret import removed from __init__.py